reproductions/StackExchange.Redis.StackOverflow/Program.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.ClrProfiler;
+
+namespace StackExchange.Redis.StackOverflow
```

Comment:
Nit: Perhaps the test name could be renamed to `StackExchange.Redis.StackOverflowException` so as not be confused with `stackoverflow.com`

Reply:
Good idea. Also renamed `HttpMessageHandler.StackOverflow` to match. e9c441a19700253f10fb0f3045f4eddf8275666c